apps/ecommerce/frontend/src/components/Dialog/ResourceList.tsx

```diff
@@ -0,0 +1,31 @@
+import { Flex } from '@contentful/f36-components';
+import ResourceCard from './ResourceCard/ResourceCard';
+import { styles } from './Dialog.styles';
+import { ExternalResource } from 'types';
+
+interface Props {
+  externalResources: ExternalResource[];
+  resourceProvider: string;
+  resourceType: string;
+}
+
+const ResourceList = (props: Props) => {
+  const { externalResources, resourceProvider, resourceType } = props;
+
+  return (
+    <Flex className={styles.productList}>
+      {externalResources?.map((item, index) => {
```

Comment:
No need for the nullable `?` operator here as externalResources has to have a value.